#! /bin/sh /usr/share/dpatch/dpatch-run
## 03_typo.dpatch by Christian Marillat <marillat@debian.org>
##
## All lines beginning with ## DP:' are a description of the patch.
## DP: 

@DPATCH@

--- mpeg4ip-1.6.orig/server/mp4live/audio_l16.cpp	2007-09-24 16:22:24.000000000 +0200
+++ mpeg4ip-1.6/server/mp4live/audio_l16.cpp	2007-09-24 16:22:35.000000000 +0200
@@ -175,7 +175,7 @@
   m_pFrameBuffer = (u_int16_t*)Malloc(m_frameBufferLength);
   
 #ifdef WORDS_BIGENDIAN
-  memcpy(m_pFrameBuffer, pSameples, m_frameBufferLength);
+  memcpy(m_pFrameBuffer, pSamples, m_frameBufferLength);
 #else
   for (uint32_t ix = 0; ix < numSamplesPerChannel * m_audioDstChannels; ix++) {
     m_pFrameBuffer[ix] = B2N_16(pSamples[ix]);
